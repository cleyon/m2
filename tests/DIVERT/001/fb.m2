@divert 1
Bueller?
I think he's sick, sir.
@divert
Good morning class.
@@ @m2ctl dump_block 1
OK
@undivert 1
@undivert 1
Class dismissed.
