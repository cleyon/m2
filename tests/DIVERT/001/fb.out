Good morning class.
OK
Bueller?
I think he's sick, sir.
Class dismissed.
